.github/workflows/workflow.yml

```diff
@@ -270,7 +270,6 @@ jobs:
         DOCKER_HOST: ssh://linkerd-docker
       run: |
         export PATH="`pwd`/bin:$PATH"
-        bin/docker image prune -f
```

Comment:
just to confirm -- this will only remove dangling images? specifically: it will not interfere with other concurrent builds?

Reply:
Yes, those layers that have no relationship to other tagged images, in theory. In practice I observed some new weird `kind` errors appearing, so I backed out of this cleanup finally. I'll document it in the packet docs as a "maintenance item" to keep out of the build.